App.js

```diff
@@ -1,105 +1,26 @@
-import React, { useRef, useEffect } from 'react';
+import React from 'react';
 import {
   SafeAreaView,
-  StyleSheet,
-  ScrollView,
   View,
   Text,
-  StatusBar,
   Image,
-  Animated,
-  FlatList,
 } from 'react-native';
+import { AppearAnim } from './src/AppearAnim.js';
+import { TodoItems} from './src/TodoItems.js';
+import { styles } from './src/styles.js';
 
-import {
-  Header,
-  Colors,
-} from 'react-native/Libraries/NewAppScreen';
-
-const ImageAnim = (props) => {
-  const imageAnim = useRef(new Animated.Value(0)).current
-
-  React.useEffect(() => {
-    Animated.timing(
-      imageAnim, {
-        toValue: 1,
-        duration: 50000,
-      }
-    ).start()
-
-  }, [])
-
-  return (
-    <Animated.View style={{...props.style,
-    opacity: imageAnim}}>
-      {props.children}
-    </Animated.View>
-  )
-}
-
-const TodoItems = () => {
+export const App: () => React$Node = () => {
   return (
-    <View style={styles.sectionContainer}>
-      <FlatList
-        data={[
-          {key: 'Cleaning'},
-          {key: 'Studying'},
-          {key: 'Read 40 pg of the Book'},
-          {key: 'Go for a walk with doggo'},
-          {key: 'Call mum'},
-          {key: 'Make dinner'},
-          {key: 'Exercise'},
-
-        ]}
-        renderItem={({item}) => <Text style={styles.highlight}>{item.key}</Text>}
-      />
-    </View>
-  );
-}
-
-const App: () => React$Node = () => {
-  return (
-    <>
       <SafeAreaView>
-      
           <View style={styles.body}>
             <View style={styles.sectionContainer}>
               <Text style={styles.sectionTitle}>Заголовок</Text>
-              <ImageAnim style={{alignItems: 'center'}}>
+              <AppearAnim style={{alignItems: 'center'}}>
```

Comment:
Inline style here but in eslint config it is specified that error should appear for inline styles